File: msg-sentiment/src/main/java/dev/number6/sentiment/ChannelMessagesSentimentRequestHandler.java
package dev.number6.sentiment;

import dev.number6.message.ChannelMessages;
import dev.number6.message.ChannelMessagesHandler;
import dev.number6.message.ChannelMessagesNotificationRequestHandler;

public class ChannelMessagesSentimentRequestHandler extends ChannelMessagesNotificationRequestHandler {

    public ChannelMessagesSentimentRequestHandler() {
        super(new ChannelMessagesHandler() {
            @Override
            public void handle(ChannelMessages channelMessages) {

            }
        });
    }
}

File: msg-sentiment/src/main/java/dev/number6/sentiment/ChannelMessagesSentimentRequestHandler.kt
package dev.number6.sentiment

import dev.number6.message.ChannelMessagesNotificationRequestHandler
import dev.number6.sentiment.dagger.DaggerChannelMessagesSentimentComprehensionComponent

class ChannelMessagesSentimentRequestHandler : ChannelMessagesNotificationRequestHandler(DaggerChannelMessagesSentimentComprehensionComponent.create().getChannelMessagesHandler())